.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules
 release
 lib
 coverage
+.vscode
```

Comment:
This should be in your global `.gitignore` file, because this is specific to your own machine.